pkg/utils/env/env.go

```diff
@@ -248,39 +240,62 @@ func DeleteEnv(envName string) (string, error) {
 		err = fmt.Errorf("you can't delete current using environment %s", curEnv)
 		return message, err
 	}
-	envdir, err := system.GetEnvDir()
-	if err != nil {
-		return message, err
-	}
-	envPath := filepath.Join(envdir, envName)
-	if _, err := os.Stat(envPath); err != nil {
-		if os.IsNotExist(err) {
-			err = fmt.Errorf("%s does not exist", envName)
-			return message, err
-		}
-	}
-	if err = os.RemoveAll(envPath); err != nil {
-		return message, err
-	}
-	message = envName + " deleted"
-	return message, err
+	err = cli.Delete(context.Background(), &v1alpha2.Environment{ObjectMeta: metav1.ObjectMeta{
+		Name: envName,
+	}})
+	return envName + " deleted", err
```

Comment:
```suggestion
	return fmt.Sprintf("environment \"%s\" deleted", envName), err
```
This looks more clear:)